src/ISA/ISA/ArcTypes/ArcTableAux.fs

```diff
@@ -275,6 +278,16 @@ module Unchecked =
                 for missingColumn,missingRow in missingKeys do
                     setCellAt (missingColumn,missingRow,empty) values
 
+    /// Increases the table size to the given new row count and fills the new rows with the last value of the column
```

Comment:
Maybe combine with `fillMissingCells` and add a flag to use empty or last value

Reply:
Now that I think about it, it doesn't really match. ExtendRows does not fill up empty cells in a previously sparse table, but extends the table to a new given rowCount.